### PR TITLE
Add installation instructions and token input

### PR DESCRIPTION
**Pull request by Hellebore**

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+
+# Sourcery VS Code Client
```
Comment (brendanator) [inline INSTALL.md]: Sourcery extension

File: INSTALL.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+This extension is currently in beta - please raise any problems or feedback as
+issues.
+
+# Installation
+
+1. Go to https://sourcery.ai/download to get a free token and copy it to your 
+   clipboard.
+2. Enter the token into the box above, or search for `sourcery` in the 
+   VS Code settings and enter the token into the `Sourcery Token` field.
```
Comment (brendanator) [inline INSTALL.md]: Remove `VS Code`

File: INSTALL.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+# How it works
+
+Sourcery scans the Python file that you have open in your editor. Refactoring 
+suggestions are underlined and listed in the Problems window. 
+
+You can hover over a suggestion to see a diff of the changes.
+
+To implement the suggestion use the Quick Fix from either the Problems window
+or the lightbulb in the code editor.
```
Comment (brendanator) [inline INSTALL.md]: - The lightbulb is only in the Problems window
- Lightbulb appears on all problems not just sourcery ones

File: INSTALL.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+
```
Comment (brendanator) [inline INSTALL.md]: Is the extra newline needed for visibility?

File: INSTALL.md (hunk 3 above)
Comment (brendanator) [reply INSTALL.md]: Presume the README.md needs updating too

File: INSTALL.md (hunk 4 above)
Comment (Hellebore) [reply INSTALL.md]: yes to avoid being hidden by the enter token box

File: INSTALL.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+# Sourcery VS Code Client
+
+Sourcery is an AI-powered coding assistant which helps you write better Python
+code faster. It works by providing refactoring suggestions on the fly that you
+can instantly integrate into your code.
+
+This extension is currently in beta - please raise any problems or feedback as
+issues.
```
[the remark this answers is not in the document]
Comment (Hellebore) [reply INSTALL.md]: done

File: INSTALL.md (hunk 1 above)
Comment (Hellebore) [reply INSTALL.md]: done

File: INSTALL.md (hunk 2 above)
Comment (Hellebore) [reply INSTALL.md]: done

File: INSTALL.md (hunk 3 above)
Comment (Hellebore) [reply INSTALL.md]: addressed in next version of binary

**Pull request merged**